config/initializers/cors.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
```

Comment:
# Frontend review

- [ ] As a registered user, I can't reserve packages:

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/814d9317-4f05-4493-90c9-8c8f3b1a5777)

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/5f83793d-7bdb-4b3e-9d9e-3d80a67e989f)

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/4c1bd51f-d2d8-4605-b8e8-5e92fdc73656)

Kindly, fix this bug so your app works as intended.  If this is not a bug, this means you app needs special procedures to be run correctly.  If this is the case, kindly add those instructions to your readme file. 👍 

- [ ] The admin user that has been included cant' login on my end using the credentials provided on seeds.rb:

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/d89da179-6d35-4876-8334-d2d82e161fd6)

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/33f9aee3-7436-4743-b0ff-75e05c18a576)

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/6e5c9b69-8acd-49b9-a189-80a3d39816a1)

I created another admin in the console, but couldn't access the functionality.

If there is a special way to give a user admin permissions, kindly add them to the readme file. 👍 

- [ ] I can't find a way to add or remove packages in the frontend:

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/ed75924e-97a8-4f7c-af4b-0cc059857bfd)

These are core features for groups of 3.  If these features exist, kindly make them more obvious in your frontend so your users can access them.

If there your project requires special procedures to be run correctly, kindly add those instructions to your readme. 👍 

- [ ] There is a login issue that is prevalent across many server requests:

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/449e0c8d-465b-40ae-9878-805005cebd82)

This could be the reason for the other issues, kindly take a look. 👍